builtin/credential/ldap/path_login.go

```diff
@@ -46,6 +47,13 @@ func pathLogin(b *backend) *framework.Path {
 
 func (b *backend) pathLoginAliasLookahead(ctx context.Context, req *logical.Request, d *framework.FieldData) (*logical.Response, error) {
 	username := d.Get("username").(string)
+	cfg, err := b.Config(ctx, req)
+	if err != nil {
+		return nil, err
+	}
+	if !*cfg.CaseSensitiveNames {
```

Comment:
This looks like another potential nil pointer dereference. You will want to guard against this runtime error with a nil check.